sylvia/tests/generics.rs

```diff
@@ -29,23 +29,166 @@ pub mod cw1 {
     }
 }
 
+pub mod whitelist {
+    use cosmwasm_std::{CosmosMsg, CustomMsg, CustomQuery, Response, StdError};
+
+    use serde::de::DeserializeOwned;
+    use serde::Deserialize;
+    use sylvia::types::{ExecCtx, QueryCtx};
+    use sylvia_derive::interface;
+
+    #[interface(module=msg)]
+    pub trait Whitelist<Msg, QueryRet>
+    where
+        for<'msg_de> Msg: CustomMsg + Deserialize<'msg_de>,
+        for<'msg_de> QueryRet: CustomQuery + DeserializeOwned,
```

Comment:
I think this `<'msg_de>` is not needed. Or maybe you forgot `+ 'msg_de`?

Reply:
it's not needed